packages/storyshots/README.md

```diff
@@ -97,3 +97,7 @@ initStoryshots({
 ```
 
 Here is an example of [a regex](https://regex101.com/r/vkBaAt/2) which does not pass if `"Relay"` is in the name: `/^((?!(r|R)elay).)*$/`.
+
+### `noMatchSnapshots`
```

Comment:
@tmeasday What do you think about creating an option `compareSnapshots` that defaults to `true`? I don't like the double negative of `noMatchSnapshots`. Or just call the option you added `skipComparison`?